ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -285,6 +285,16 @@ As of AVD 5.0.0 "null" option for esp integrity and encryption has been replaced
 +      encryption: disabled
 ```
 
+### Added valid for data models `router_bgp.address_family_ipv4_multicast.redistribute_routes[].source_protocol`, `router_bgp.address_family_ipv6.redistribute_routes[].source_protocol`, and `router_bgp.vrf.redistribute_routes[].source_protocol`
+
+Added valid for data models `router_bgp.address_family_ipv4_multicast.redistribute_routes[].source_protocol`, `router_bgp.address_family_ipv6.redistribute_routes[].source_protocol`, and `router_bgp.vrf.redistribute_routes[].source_protocol` to restrict the source protocol of redistribute routes in EOS configuration.
```

Comment:
```suggestion
### Added valid values for data models `router_bgp.address_family_ipv4_multicast.redistribute_routes[].source_protocol`, `router_bgp.address_family_ipv6.redistribute_routes[].source_protocol`, and `router_bgp.vrf.redistribute_routes[].source_protocol`

Added valid values for data models `router_bgp.address_family_ipv4_multicast.redistribute_routes[].source_protocol`, `router_bgp.address_family_ipv6.redistribute_routes[].source_protocol`, and `router_bgp.vrf.redistribute_routes[].source_protocol` to restrict the source protocol of redistribute routes in EOS configuration.
```

Reply:
We can brief line 288 like - Added valid values for `redistribute_routes[].source_protocol` in `router_bgp` data model.